packages/server/email/components/SummaryEmail/MeetingSummaryEmail/SummarySheet.tsx

```diff
@@ -34,9 +33,8 @@ const sheetStyle = {
 }
 
 const SummarySheet = (props: Props) => {
-  const {emailCSVUrl, urlAction, meeting, meetingUrl, referrer, teamDashUrl} = props
```

Comment:
looks like teamName & meetingUrl are no longer used. if so could you remove from the graphql query & Props interface?

Reply:
should i remove createdAt and meetingNumber from the graphql query as well?

Reply:
yep!